crux/src/Crux.hs

```diff
@@ -287,3 +289,25 @@ runSimulator lang opts@(cruxOpts,_) =
        writeProf profInfo
 
      readIORef gls
+
+reportStatus :: (?outputConfig::OutputConfig) => ProvedGoals a -> IO ()
```

Comment:
We report "Overall status: Valid" even if simulation timed out.  It seems like incomplete would be a better verdict.

Reply:
Good point. I'll change that.

Reply:
Fixed in e4dd45f.